x-pack/plugin/esql/qa/testFixtures/src/main/resources/ip.csv-spec

```diff
@@ -317,7 +317,7 @@ FROM logs
 2023-10-23T13:55:01.546Z|          java|More java stuff                    |null
 ;
 
-toIpInAgg
+toIpInAgg#[skip:-8.12.99,reason:StatsNestedExp introduced in v8.13.0]
```

Comment:
This relies on https://github.com/elastic/elasticsearch/pull/104387 added in 8.13